unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Animator/DCLAnimator.cs

```diff
@@ -47,8 +47,8 @@ public class DCLAnimationState
 
```

Comment:
You forgot to unsubscribe from the proper method (can't reference the code because is not included in the changeset).

This is in the current code:
`private void OnDestroy() { entity.OnShapeUpdated -= OnComponentUpdated; }`

It should be:
`private void OnDestroy() { entity.OnShapeLoaded -= OnComponentUpdated; }`

Reply:
Good catch!